library/Fission/User/Creator/Error.hs

```diff
@@ -18,3 +21,16 @@ instance Display AlreadyExists where
 instance ToServerError AlreadyExists where
   toServerError AlreadyExists =
     err409 { errBody = UTF8.showLazyBS <| textDisplay AlreadyExists }
+
+data InvalidUsername = InvalidUsername
```

Comment:
Do we still need this type if we have `Username.Invalid`? Doesn't seem to be used anywhere

Reply:
Good catch! 🔪